lib/networkPacks/PacksForLobby.h

```diff
@@ -381,4 +383,22 @@ struct DLL_LINKAGE LobbyPvPAction : public CLobbyPackToServer
 	}
 };
 
+struct DLL_LINKAGE LobbyDelete : public CLobbyPackToServer
+{
+	enum EType : ui8 {
```

Comment:
To prevent accidental int<->enum conversions
```suggestion
	enum class EType : ui8 {
```